test/blackbox-tests/test-cases/coq/theory-stanza-duplicate-module.t/run.t

```diff
@@ -7,60 +7,9 @@ Dune should warn about duplicate modules declared inside a coq.theory stanza
   > EOF
 
   $ dune build
-  Internal error, please report upstream including the contents of _build/log.
-  Description:
-    ("Map.add_exn: key already exists",
-    { key =
-        { source = In_build_dir "default/foo.v"; prefix = []; name = "foo" }
-    })
-  Raised at Stdune__Code_error.raise in file "otherlibs/stdune/code_error.ml",
```

Comment:
Let's merge this ASAP because we don't want stack traces in the test suite. Stack traces will change due to unrelated modifications in the code so we should never commit them.